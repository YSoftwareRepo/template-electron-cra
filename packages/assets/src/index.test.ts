import { Assets } from './index';

test('Test exports of assets', () => expect(Assets).toMatchSnapshot());
