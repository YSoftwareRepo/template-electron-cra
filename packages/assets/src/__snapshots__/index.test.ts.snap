// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test exports of assets 1`] = `
Object {
  "logo": "logo.svg",
}
`;
